### PR TITLE
STOR-829: set security.openshift.io/csi-ephemeral-volume-profile label

### PR DESCRIPTION
**Pull request by dobsonj**
https://issues.redhat.com/browse/STOR-829
Enhancement: https://github.com/openshift/enhancements/blob/master/enhancements/storage/csi-inline-vol-security.md
/hold for https://github.com/openshift/kubernetes/pull/1384 in case the label changes
/cc @openshift/storage

### CODE REVIEW

**Comment by dobsonj** [comment]
/assign @coreydaley 
for `/approve`

/assign @Phaow 
for `/label qe-approved`
/assign @lpettyjo 
for `/label docs-approved`
/assign @mhackett
for `/label px-approved`

**Comment by Phaow** [comment]
/label qe-approved
Pre merged test passed, **[details test record](https://drive.google.com/file/d/1OfIdWWIgO-MGFjH2qHcEAXEDFbrtT_Gz/view?usp=sharing)**.

**Comment by dobsonj** [comment]
one failure: `[sig-storage] Volume limits should verify that all nodes have volume limits [Skipped:NoOptionalCapabilities] [Suite:openshift/conformance/parallel] [Suite:k8s] `
/retest

**Comment by dobsonj** [comment]
Apparently the 1 test that is failing was removed from the release-1.25 branch upstream:
https://github.com/kubernetes/kubernetes/commit/ea903c6219cd38b9bc55804df5c0e8780bd830b4

**Comment by jsafrane** [comment]
/override ci/prow/e2e-aws-techpreview
trying my power over this repo

**Comment by dobsonj** [comment]
@gabemontero @coreydaley could either of you override `ci/prow/e2e-aws-techpreview`?
We now have some [failing TechPreview tests](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_vmware-vsphere-csi-driver/49/pull-ci-openshift-vmware-vsphere-csi-driver-master-e2e-vsphere-csi-migration/1597342389087768576) until this PR merges.

**Comment by gabemontero** [comment]
hmmm ... does not look like I have that authority either

perhaps an openshift/release PR to make the job optional, at least to get this merged, and then revert, is needed at this point

**Comment by dobsonj** [comment]
@gabemontero it did work! Thanks a lot, the bot was just slow :)

**Comment by gabemontero** [comment]
ah it was just super slow :-)

File: assets/csidriver.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ metadata:
   annotations:
     # This CSIDriver is managed by an OCP CSI operator
     csi.openshift.io/managed: "true"
+  labels:
+    security.openshift.io/csi-ephemeral-volume-profile: "restricted"
```
Comment (gabemontero) [inline assets/csidriver.yaml:9]: so to follow up on our conversation this morning @coreydaley @adambkaplan @siamaksade @rgormley the storage team already set the label on our driver that per https://github.com/openshift/jenkins-sync-plugin/blob/master/hack/tag-ci-image.sh means anybody who is a `restricted` user or higher can use the shared resource driver.

**Pull request merged**